client/lib/screens/main_screen.dart

```diff
@@ -27,16 +32,40 @@ class MainScreen extends StatelessWidget {
                 showSearch(context: context, delegate: videoSearchDelegate);
               },
             ),
-            Consumer<CustomTheme>(
-              builder: (context, theme, _child) => IconButton(
-                  key: const ValueKey("icon-theme-toggle-button"),
-                  onPressed: () => theme.toggleMode(),
-                  icon: theme.icon),
-            )
+            const SizedBox(width: 5,),
+            PopupMenuButton(
+                itemBuilder: (BuildContext context) => <PopupMenuEntry<VertMenu>>[
+                  CheckedPopupMenuItem<VertMenu>(
+                    value: VertMenu.themeMode,
+                    checked: context.read<CustomTheme>().isDarkMode,
+                    child: const Center(child: Text('다크모드')),
```

Comment:
Center는 왜 넣으셨나요?

```suggestion
                    child: Text('다크모드'),
```

Center 는 일관성이 안맞기 때문에